Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/MaterialComponentController.cpp

```diff
@@ -788,9 +806,18 @@ namespace AZ
             }
         }
 
-        void MaterialComponentController::QueueMaterialUpdateNotification()
+        void MaterialComponentController::QueueMaterialsCreatedNotification()
+        {
+            m_queuedMaterialsCreatedNotification = true;
+            if (!SystemTickBus::Handler::BusIsConnected())
+            {
+                SystemTickBus::Handler::BusConnect();
+            }
```

Comment:
nit: Doesn't `BusConnect()` already check internally for `BusIsConnected()`?

Reply:
Yes it does indeed!